jxl/src/api/inner/codestream_parser/sections.rs

```diff
@@ -71,6 +71,10 @@ impl CodestreamParser {
             frame.prepare_render_pipeline()?;
             frame.finalize_lf()?;
             frame.decode_hf_group(0, 0, &mut br)?;
+            // Clear available_sections after processing the single-group special case.
+            // Without this, the assertion at line 160 would fail because available_sections
```

Comment:
Please remove this sentence from the comment, or at least remove the line number.

Reply:
Done